### PR TITLE
add http4k

### PR DESCRIPTION
**Pull request by jamesward**

### CODE REVIEW

**Comment by bshaffer** [comment]
@jamesward for starters can we add this to [the github actions tests.yaml file](https://github.com/GoogleCloudPlatform/kotlin-samples/blob/master/.github/workflows/tests.yml#L109)?

**Review by bshaffer — COMMENTED** [review]
@jamesward for starters can we add this to [the github actions tests.yaml file](https://github.com/GoogleCloudPlatform/kotlin-samples/blob/master/.github/workflows/tests.yml#L109)?

**Comment by jamesward** [comment]
Done.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -87,6 +87,13 @@ jobs:
         build-root-directory: run/grpc-hello-world-streaming
         wrapper-directory: run/grpc-hello-world-streaming
 
+    - name: Cloud Run (gRPC bidi-streaming) - Gradle
+      uses: eskatos/gradle-command-action@v1
+      with:
+        arguments: test --console=plain --info
+        build-root-directory: run/grpc-hello-world-bidi-streaming
+        wrapper-directory: run/grpc-hello-world-bidi-streaming
```
Comment (bshaffer) [inline .github/workflows/tests.yml:95]: 🥂 I was hoping you'd add this one!!

File: run/http4k-hello-world/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+1. Enable the Container, Container Registry, Cloud Build, and Cloud Run APIs:
+    ```
+    gcloud services enable container.googleapis.com containerregistry.googleapis.com cloudbuild.googleapis.com run.googleapis.com
+    ```
+
+1. Build the container image on Cloud Build using Buildpacks (currently an alpha feature), storing the image on Google Container Registry:
+    ```
+    export PROJECT_ID=YOUR_GCP_PROJECT
+    gcloud alpha builds submit --pack=image=gcr.io/$PROJECT_ID/http4k-hello-world
```
Comment (bshaffer) [inline run/http4k-hello-world/README.md:25]: is `alpha` intentional here?

**Review by bshaffer — APPROVED** [review]
Tested local, docker local, and CLI deploy on Cloud Run. Worked like a charm. Nice job!

... one question, otherwise approved 👍

File: run/http4k-hello-world/README.md (hunk 2 above)
Comment (jamesward) [reply run/http4k-hello-world/README.md:25]: Just a copy & paste thing. Will resolve with #117

**Pull request merged**